src/work-allocation-2/work-allocation-feature.routes.ts

```diff
@@ -49,6 +50,10 @@ export const ROUTES: Routes = [
             data: {
               title: 'HMCTS Manage cases | My work | Available tasks', subTitle: 'Available tasks'
             }
+          },
+          {
+            path: 'find-person',
+            component: FindPersonContainerComponent
```

Comment:
if this is a common component, do we need it in a route?

Reply:
Again this route will be deleted once @sreekanthpuligadda  tests it. It is only for TESTING. I will delete it once its TESTED

Reply:
We could get it tested in the final e2e journey, don't need to add this for test purpose of a common component.

Reply:
This will not be merged. I will delete it before the merge.

Reply:
as agreed this will be deleted before merging